samples/drivers/led_max7219/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  description: Demonstration of the MAX7219 LED driver
+  name: MAX7219 sample
+tests:
+  sample.drivers.led.max7219:
+    platform_allow: stm32_min_dev_black
```

Comment:
please filter based on `dt_compat_enabled`

Reply:
I will add it. Thanks.